src/Extensions/Features/src/FeatureCollection.cs

```diff
@@ -117,6 +117,16 @@ IEnumerator IEnumerable.GetEnumerator()
     /// <inheritdoc />
     public TFeature? Get<TFeature>()
     {
+        if (typeof(TFeature).IsValueType)
+        {
+            var feature = this[typeof(TFeature)];
+            if (feature is null && Nullable.GetUnderlyingType(typeof(TFeature)) is null)
+            {
+                throw new InvalidOperationException($"{typeof(TFeature).FullName} does not exist in the feature collection " +
+                $"and because it is a struct we can't return null. Use 'featureCollection[typeof({typeof(TFeature).FullName})] is not null' to check if it exists.");
```

Comment:
```suggestion
                throw new InvalidOperationException(
                    $"{typeof(TFeature).FullName} does not exist in the feature collection " +
                    $"and because it is a struct the method can't return null. Use 'featureCollection[typeof({typeof(TFeature).FullName})] is not null' to check if the feature exists.");
```

Reply:
"we can't return null" in the exception is unusual. Changed it to "the method can't return null"